DOC-2181: TinyMCE 6.8.1 release notes. (#2956)
tinymce/tinymce-docs: modules/ROOT/pages/6.8.1-release-notes.adoc

= TinyMCE 6.8.1

> :navtitle: TinyMCE 6.8.1

Reviewer: Since we never released 6.8.0 shouldn't this include all the items for 6.8.0 as well? Such as the new `default_font_stack` option?

Author: @spocke the release notes have all tickets in `scope` for `6.8.0` and `6.8.1`, but the confusion was that i thought you had merged a PR for the new `default_font_stack` option, but just remembered that you created a `draft` in the Jira ticket.  I will put this together soon- thanks for catching it early.

Author: Added in [this pr](https://github.com/tinymce/tinymce-docs/pull/3021), for new `default_font_stack` option in the `6.8.1` release notes.

=== Bespoke dropdown toolbar buttons including align, fontfamily, fontsize, blocks, and styles did not include their visible text labels in their accessible names
// #TINY-10147
Prior to {productname} 6.8.1, accessibility labels of bespoke dropdown buttons only announced the name of the button, and not the visible text label of the selected option.

For example, when the `fontfamily` dropdown button selection was set to “Verdana”, the accessibility name was set as “Fonts”. However, it should have been set as “Fonts Verdana” as the selected option is also visible on the button.

In {productname} 6.8.1, the visible text label _(or icon name where icons are present instead)_ of selected options of bespoke dropdown buttons are appended to the accessible name. This ensures that the accessible name of the above scenario is announced as “Fonts Verdana”, improving accessibility.

[[additions]]
== Additions

{productname} 6.8.1 also includes the following additions:

=== Added `force_hex_color` editor option. Option `'always'` converts all RGB & RGBA colours to hex, `'rgb_only'` will only convert RGB and *not* RGBA colours to hex, `'off'` won't convert any colours to hex
// #TINY-9819

> Previously, when applying colors to content within the editor via the `forecolor` and `backcolor`, only colors applied with RGB values were excepted.
> 
> As a consequence HEX colors were ignored by default in {productname}.

Author: ```suggestion
Previously, when applying colors to content within the editor via the `forecolor` and `backcolor`, HEX colors were always converted to RGB.
```

> As a consequence HEX colors were ignored by default in {productname}.

Author: thanks for pointing this out @spocke, I have updated the content after discussions with @danoaky-tiny

{productname} 6.8.1 addresses this issue, by adding the new editor option `force_hex_color: 'off'` which is set as **default**, can now be added to the {productname} config by;

> * **force_hex_color: 'always'**: all colors are converted and applied as HEX format.

Author: thanks @spocke , we have added this to the options as requested.
